views/about.lavender

```diff
@@ -5,12 +5,30 @@ head
   meta(name='keywords', content='Lavender, Lavender programming language, lavenderlang, lavender-lang')
   meta(name='description', content='A templating engine for PHP, designed to make your life easier.')
   meta(name='viewport', content='width=device-width, initial-scale=1')
+  link(rel="apple-touch-icon", sizes="57x57", href="/img/favicons/apple-touch-icon-57x57.png")
+  link(rel="apple-touch-icon", sizes="60x60", href="/img/favicons/apple-touch-icon-60x60.png")
+  link(rel="apple-touch-icon", sizes="72x72", href="/img/favicons/apple-touch-icon-72x72.png")
+  link(rel="apple-touch-icon", sizes="76x76", href="/img/favicons/apple-touch-icon-76x76.png")
+  link(rel="apple-touch-icon", sizes="114x114", href="/img/favicons/apple-touch-icon-114x114.png")
+  link(rel="apple-touch-icon", sizes="120x120", href="/img/favicons/apple-touch-icon-120x120.png")
+  link(rel="apple-touch-icon", sizes="144x144", href="/img/favicons/apple-touch-icon-144x144.png")
+  link(rel="apple-touch-icon", sizes="152x152", href="/img/favicons/apple-touch-icon-152x152.png")
+  link(rel="apple-touch-icon", sizes="180x180", href="/img/favicons/apple-touch-icon-180x180.png")
+  link(rel="icon", type="image/png", href="/img/favicons/favicon-32x32.png", sizes="32x32")
+  link(rel="icon", type="image/png", href="/img/favicons/favicon-194x194.png", sizes="194x194")
+  link(rel="icon", type="image/png", href="/img/favicons/favicon-96x96.png", sizes="96x96")
+  link(rel="icon", type="image/png", href="/img/favicons/android-chrome-192x192.png", sizes="192x192")
+  link(rel="icon", type="image/png", href="/img/favicons/favicon-16x16.png", sizes="16x16")
+  link(rel="manifest", href="/img/favicons/manifest.json")
+  meta(name="msapplication-TileColor", content="#cdb4cc")
+  meta(name="msapplication-TileImage", content="/img/favicons/mstile-144x144.png")
+  meta(name="theme-color", content="#796878")
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Oswald:400,700')
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Lato:300,700')
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Special+Elite')
```

Comment:
You can combine the google fonts stylesheets using a pipe.

http://fonts.googleapis.com/css?family=Oswald:400,700|Lato:300,700|Special+Elite